pkg/doc/cm/credentialmanifest.go

```diff
@@ -137,27 +136,68 @@ func (cm *CredentialManifest) UnmarshalJSON(data []byte) error {
 // Validate ensures that this CredentialManifest is valid as per the spec.
 // Note that this function is automatically called when unmarshalling a []byte into a CredentialManifest.
 func (cm *CredentialManifest) Validate() error {
-	if cm.Issuer.ID == "" {
-		return errors.New("issuer ID missing")
+	if cm.ID == "" {
+		return errors.New("ID missing")
+	}
+
+	err := validateIssuer(cm.Issuer)
+	if err != nil {
+		return err
 	}
 
 	if len(cm.OutputDescriptors) == 0 {
 		return errors.New("no output descriptors found")
 	}
 
-	err := ValidateOutputDescriptors(cm.OutputDescriptors)
+	err = ValidateOutputDescriptors(cm.OutputDescriptors)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
+func validateIssuer(issuer Issuer) error {
+	if issuer == (Issuer{}) {
```

Comment:
A better way to check this condition would be to make the `Issuer` field a pointer, rather than doing a struct value comparison like this. Strictly speaking, it doesn't make sense to say a non-pointer struct is "missing", since you don't know whether it was just defaulting to to the zero values or someone (for some reason) purposefully initialized it with zero values.

Alternatively, you might consider simply removing this check. Is there a meaningful difference between the entire `issuer` struct being missing and the issuer ID being missing?

Reply:
Oh that's true, I'll remove this check!